fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -2796,6 +2800,30 @@ private void removeHelperNode(String host, int port) {
         helperNodes.removeIf(node -> node.getHost().equals(host) && node.getPort() == port);
     }
 
+    private void ensureSafeToDropAliveFollower() throws DdlException {
+        int numFollower = 0;
+        int numAliveFollower = 0;
+        for (Frontend fe : frontends.values()) {
+            if (fe.getRole() == FrontendNodeType.FOLLOWER) {
+                numFollower += 1;
+                if (fe.isAlive()) {
```

Comment:
Alive status is set by heart mgr,  the latency time is  5 seconds,  In some situation, it can not detect fe is down ?